debian/scripts/unbundle

```diff
@@ -8,7 +8,7 @@ sys.path.append("build/linux/unbundle")
 
 import replace_gn_files
 
-keepers = ('re2')
```

Comment:
BTW since I saw this in another PR, I should point out that this is interpreted as a string, not a tuple containing a single item. If you want a tuple of a single item, you'd want to include a comma, such as: `('re2', )` (space is not necessary; only for readability). Alternatively, you could just use a list: `['re2']`

Reply:
Ok. The current upstream version has an empty set of ()s assigned to keepers so it shouldn't be a problem for this branch. I'll look at Bionic.